GVRf/Sample/gvr-video/src/org/gearvrf/video/VideoScript.java

```diff
@@ -410,6 +410,12 @@ public void run() {
                     mGVRContext, "button/play-active.png"));
             mPlayPauseButton = new GVRSceneObject(gvrContext,
                     gvrContext.createQuad(0.7f, 0.7f), mInactivePause);
+            mPlayPauseButton.getRenderData().getMaterial().setTexture("active_play", mActivePlay);
```

Comment:
I think it may be an issue for us to write a sample that uses setKeepWrapper API(when necessary) or otherwise, developers may be confused with this.